### PR TITLE
docs: Updated the Readme.md that will also be used in extension marketplace.

### PR DESCRIPTION
**Pull request by sffamily**
[User Story 14398093](https://msazure.visualstudio.com/Microsoft%20Teams%20Extensibility/_workitems/edit/14398093): Make the VSC extension introduction page is up-to-date

### CODE REVIEW

File: packages/vscode-extension/README.md [hunk 1]
```diff
@@ -1,18 +1,17 @@
 # Microsoft Teams Toolkit for Visual Studio Code
-
-## What is the Teams Toolkit?
+## What is  Teams Toolkit?
```
Comment (tecton) [inline packages/vscode-extension/README.md:2]: minor: double spaces here, but it looks good when rendered as Markdown.

**Review by tecton — APPROVED** [review]

File: packages/vscode-extension/README.md [hunk 2]
```diff
@@ -76,15 +75,14 @@ At its core, the Teams app embraces three components:
 
 When an app is installed, the Teams client parses the manifest file to determine needed information like the name of your app and the URL where the services are located.
 
-- To configure your app, navigate to the `appPackage/manifest.source.json` file and edit the manifest.
-- To validate your manifest file, click `Validate manifest file` in the sidebar menu.
+- To configure your app, navigate to the `appPackage/manifest.template.json` file or from UI to select `DEVELOPMENT`-`Edit manifest file` and edit the manifest.
 - The toolkit will automatically update the app registration data accordingly during app side-loading and publish.
 
 ### Preview your app on your local/remote dev environment
```
Comment (zhenjiao-ms) [inline packages/vscode-extension/README.md:82]: In each section, better to have the link to the related contents in docs site. This one can be https://docs.microsoft.com/en-us/microsoftteams/platform/toolkit/debug-local

**Review by zhenjiao-ms — CHANGES_REQUESTED** [review]

File: packages/vscode-extension/README.md (hunk 1 above)
Comment (sffamily) [reply packages/vscode-extension/README.md]: fixed

File: packages/vscode-extension/README.md [hunk 3]
```diff
@@ -51,12 +50,12 @@ Verify you have the right prerequisites for building Teams apps and install some
...
 ### Create your project
 
-Use the Teams Toolkit in Visual Studio Code to set up your first app project. Create your app project using the following steps:
+Use the Teams Toolkit for Visual Studio Code to set up your first app project. Create your app project using the following steps:
 
-- Ensure you've installed the Microsoft Teams Toolkit
-- In Visual Studio Code, in `DEVELOPMENT` section click `Create a new Teams app`. Or open the Command Palette (Ctrl+Shift+P / ⌘⇧-P or View -> Command - Palette) and type "teams" and choose `Teams: Create New Project`.
+- Ensure you've installed the Microsoft Teams Toolkit for Visual Studio Code
+- In Visual Studio Code, in `DEVELOPMENT` section click `Create a new Teams app`. Or open the Command Palette (Ctrl+Shift+P / ⌘⇧-P or View -> Command - Palette) and type "teams" and choose `Teams: Create a new Teams app`.
 - Next, Add capabilities comes up. Select capability you want then Next.
```
[the remark this answers is not in the document]
Comment (sffamily) [reply packages/vscode-extension/README.md]: used the steps from doc to update these steps.

File: packages/vscode-extension/README.md (hunk 2 above)
Comment (sffamily) [reply packages/vscode-extension/README.md:85]: good point, added the read more doc link at end of each section.

File: packages/vscode-extension/README.md [hunk 4]
```diff
@@ -51,15 +50,18 @@ Verify you have the right prerequisites for building Teams apps and install some
...
-- Choose options according to your purpose, i.e., front-end hosting type, language, cloud resources and so on.
+- Ensure you've installed the Microsoft Teams Toolkit for Visual Studio Code
+- Select the Teams Toolkit icon in the Visual Studio Code sidebar.
+- Select `Create a new Teams app` button and at next prompt to also select `Create a new Teams app`.
+- Ensure that Tab is selected as the capability that you want to build in your app. Select OK.
+- Select JavaScript as the programming language.
 - Choose a location where your new application will be created in a new folder.
 - Type a name for your project and hit Enter.
 
+read more on [creating new project](https://docs.microsoft.com/en-us/microsoftteams/platform/toolkit/create-new-project).
```
Comment (HuihuiWu-Microsoft) [inline packages/vscode-extension/README.md:63]: Use capital?

Comment (sffamily) [reply packages/vscode-extension/README.md]: makes sense. all fixed. and being processed.

File: packages/vscode-extension/README.md [hunk 5]
```diff
@@ -51,15 +50,18 @@ Verify you have the right prerequisites for building Teams apps and install some
...
+Use the Teams Toolkit for Visual Studio Code to set up your first app project. Create your tab app project using the following steps:
 
-- Ensure you've installed the Microsoft Teams Toolkit
-- In Visual Studio Code, in `DEVELOPMENT` section click `Create a new Teams app`. Or open the Command Palette (Ctrl+Shift+P / ⌘⇧-P or View -> Command - Palette) and type "teams" and choose `Teams: Create New Project`.
-- Next, Add capabilities comes up. Select capability you want then Next.
-- Choose options according to your purpose, i.e., front-end hosting type, language, cloud resources and so on.
+- Ensure you've installed the Microsoft Teams Toolkit for Visual Studio Code
+- Select the Teams Toolkit icon in the Visual Studio Code sidebar.
+- Select `Create a new Teams app` button and at next prompt to also select `Create a new Teams app`.
+- Ensure that Tab is selected as the capability that you want to build in your app. Select OK.
```
Comment (zhenjiao-ms) [inline packages/vscode-extension/README.md:58]: It should be replaced by something like "select the template based on your scenarios or required Teams capabilities", it isn't just for tab

File: packages/vscode-extension/README.md [hunk 6]
```diff
@@ -51,15 +50,18 @@ Verify you have the right prerequisites for building Teams apps and install some
...
+- Select JavaScript as the programming language.
```
Comment (zhenjiao-ms) [inline packages/vscode-extension/README.md:59]: should be "select the language you prefer"

**Review by zhenjiao-ms — CHANGES_REQUESTED** [review]

File: packages/vscode-extension/README.md (hunk 5 above)
Comment (sffamily) [reply packages/vscode-extension/README.md:58]: I added in the front that creating your first app becomes creating your first tab app, this way it is the quickest way for the 'first' app. so it is now tailored for tab. try to make the step one the simplest possible to get something run. and then then have all links to official doc. what do you think?

File: packages/vscode-extension/README.md (hunk 6 above)
Comment (sffamily) [reply packages/vscode-extension/README.md:59]: as explained. making the first section the simplest possible experience, as we've inserted the Read More link to the official user manual doc.

**Pull request merged**